handler/handler.go

```diff
@@ -0,0 +1,96 @@
+package handler
+
+import (
+	"fmt"
+	"net/http"
+
+	"echo/models"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/labstack/echo/v4"
+)
+
```

Comment:
What is the purpose of this function? you doesn't called it anywhere

Reply:
will correct the mistake